ui/app/common/components/pnc-build-metrics/pncBuildMetrics.js

```diff
@@ -236,7 +236,7 @@
      */
     var loadBuildMetrics = function(builds) {
       var buildIds = builds.map(function(build) {
-        return build.buildContentId ? build.buildContentId.toString().split('-')[1] : null;
+        return utils.getBuildName(build);
```

Comment:
these values are used for Kafka endpoint to get build metrics and Kafka expects buildContentId without 'build-' prefix ATM. So the metrics won't get loaded now. You can either contact Dustin and discuss changing this in Kafka but the build name is not unique I believe so the displayed value of the build name should be changed somewhere else probably.

Reply:
Thanks, I will check this as I wasn't sure and kafka wasn't really cooperating + few times it blip what I was expecting to be there